databases/week2/week2_part2.sql

```diff
@@ -0,0 +1,35 @@
+/*
+Part 2: School database
+*/
+
+--  Create a new database containing the following tables:
+create database school ;
+
+--  Class: with the columns: id, name, begins (date), ends (date)
+create table class(
+id int(10) NOT NUll auto_increment ,
+name varchar(45) NOT NULL,
+begin_date date,
+ends_date date,
+primary key(id)
+);
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+create table Student(
+id int(15) not null auto_increment,
+name varchar(45) not null,
+email varchar(25),
+phone int(15),
+class_id int(10)not null,
+primary key(id),
+foreign key (class_id) references class(id)
+);
+-- Create an index on the name column of the student table.
+create index index_name
+on Student(name);
```

Comment:
Give a proper name when create a index for ex. index_student
```suggestion
create index index_student
on Student(id);
```
index on a VARCHAR column will boost query performance
It's make more sense to find a INT column to create INDEX on it.